evap/staff/templates/staff_semester_view_evaluation.html

```diff
@@ -189,6 +189,40 @@
             <a href="{% url 'staff:evaluation_copy' evaluation.id %}" class="btn btn-sm btn-light" data-bs-toggle="tooltip" title="{% translate 'Copy' %}">
                 <span class="fas fa-copy"></span>
             </a>
+            {% if not evaluation.has_exam %}
+                <confirmation-modal type="submit" name="evaluation_id" value="{{ evaluation.id }}" confirm-button-class="btn-primary" form="exam_creation_form_{{ evaluation.id }}">
+                    <span slot="title">{% translate 'Create exam evaluation' %}</span>
+                    <span slot="action-text" id="submit_button">{% translate 'Create exam evaluation' %}</span>
+                    <span slot="question">
+                        {% blocktranslate trimmed %}
+                            Create an exam evaluation based on this evaluation. This will copy all the participants and contributors from the original evaluation. It will set the weight of the original evaluation to 9 and its end date will be set to the day before the exam.
+                        {% endblocktranslate %}
+                    </span>
+                    <div slot="extra-inputs">
+                        <label>
+                            {% translate 'Exam Date:' %}
+                            <input type="date" name="exam_date" data-earliest-legal-exam-date="{{ evaluation.vote_start_datetime.date() + datetime.timedelta(days=1) }}" class="form-control" required form="exam_creation_form_{{ evaluation.id }}"/>
+                        </label>
+                    </div>
+
+                    <button slot="show-button" type="button" class="btn btn-sm btn-light" title="{% translate 'Create exam evaluation' %}" data-bs-placement="top" data-bs-toggle="tooltip"><span class="fas fa-file-pen fa-fw"></span></button>
+
+                    <script type="text/javascript">
+                        document.querySelectorAll("[data-evaluation-date]").forEach(element => {
```

Comment:
Is this encapsulated in some kind of shadow root, or does this find all modals on the page, and thus create O(n^2) event listeners?

(Maybe, if we have to think about it, just make it `query` just inside the current `confirmation-modal` anyway?)

(`data-evaluation-date` seems outdated, should probably be `data-earliest-lega-exam-date` now?)